frontend/types/types.ts

```diff
@@ -65,27 +65,34 @@ export interface FileInterface {
   updatedAt: Date
 }
 
-export type ReviewComment = {
-  message: string
-  user: string
-  createdAt: string
-}
+export const ResponseKind = {
+  Review: 'review',
+  Comment: 'comment',
+} as const
+export type ResponseKindKeys = (typeof ResponseKind)[keyof typeof ResponseKind]
 
-export type ReviewResponseKind = ReviewComment | ReviewResponse
+export interface ResponseInterface {
+  entity: string
+  kind: ResponseKindKeys
+  parentId: string
+  outdated?: boolean
+  decision?: DecisionKeys
+  comment?: string
```

Comment:
`comment` is a little confusing as a property name now considering the response can itself be a comment or a review. Consider renaming to something like `message` or `text`?